packages/app/models/cow-order/cow-order.ts

```diff
@@ -1,15 +1,15 @@
-type ChainId = 1 | 100;
+import { ChainId } from "@stackly/sdk";
```

Comment:
There are others places where we are using `1 | 100`. Can you fixed those?

Reply:
Didn't find any, other were hardcoded that's related to this: 

https://github.com/SwaprHQ/stackly-ui/pull/33#discussion_r1280881022